compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/layout/WindowInsetsRulers.skiko.kt

```diff
@@ -16,12 +16,107 @@
 
 package androidx.compose.ui.layout
 
-// TODO https://youtrack.jetbrains.com/issue/CMP-8086
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.CaptionBar
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.DisplayCutout
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.Ime
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.MandatorySystemGestures
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.NavigationBars
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.StatusBars
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.SystemGestures
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.TappableElement
+import androidx.compose.ui.layout.WindowInsetsRulers.Companion.Waterfall
+import androidx.compose.ui.node.LayoutModifierNode
+import androidx.compose.ui.node.ModifierNodeElement
+import androidx.compose.ui.node.TraversableNode
+import androidx.compose.ui.platform.PlatformInsets
+import androidx.compose.ui.platform.PlatformWindowInsetsConfig
+import androidx.compose.ui.unit.Constraints
 
-internal actual fun findDisplayCutouts(placementScope: Placeable.PlacementScope): List<RectRulers> =
-    emptyList()
+internal actual fun findDisplayCutouts(placementScope: Placeable.PlacementScope): List<RectRulers> {
+    return emptyList()
```

Comment:
It looks like it should be non empty to properly support new rullers API

Reply:
I had these changes stashed, thanks for the comments. I have added the implementation, can you have a look, please?